esi_leap/db/api.py

```diff
@@ -114,6 +114,11 @@ def offer_verify_availability(offer_ref, start, end):
         offer_ref, start, end)
 
 
+def offer_verify_node_availability(resource_node_uuid, start, end):
+    return IMPL.offer_verify_availability(
```

Comment:
I think you meant the new verify function here.

Reply:
fixed